src/http/client.cr

```diff
@@ -13,6 +13,20 @@
 # response.body.lines.first # => "<!doctype html>"
 # ```
 #
+# ### Parameters
+#
+# Parameters can be added to any request with the `HTTP::Params#encode` method, which
+# converts a `Hash` or `NamedTuple` to a URL encoded HTTP query.
+#
+# ```
+# require "http/client"
+#
+# params = HTTP::Params.encode({ "q" => "test"}) # => q=test
+# response = HTTP::Client.get "http://www.example.com?" + params
+# response.status_code      # => 200
+# response.body.lines.first # => "<!doctype html>"
```

Comment:
No need to repeat this from the previous example if it's not relevant.

Reply:
I think it'd be a good idea to keep at least one of those output lines to show symmetry with the previous example.